<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PiercingPointDialog</class>
 <widget class="QDialog" name="PiercingPointDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find piercing point</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QtAddons::ImageViewerWidget" name="viewer" native="true">
       <property name="minimumSize">
        <size>
         <width>386</width>
         <height>386</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Position</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_posX">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="doubleSpinBox_posY">
           <property name="maximum">
            <double>5000.000000000000000</double>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QCheckBox" name="checkBox_gainCorrection">
           <property name="text">
            <string>Correct gain</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QCheckBox" name="checkBox_useROI">
           <property name="text">
            <string>Use ROI</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_x0">
           <property name="text">
            <string>x0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_roi0">
           <property name="maximum">
            <number>5000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_y0">
           <property name="text">
            <string>y0</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_roi1">
           <property name="maximum">
            <number>5000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_x1">
           <property name="text">
            <string>x1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_roi2">
           <property name="maximum">
            <number>5000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_y1">
           <property name="text">
            <string>y1</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_roi3">
           <property name="maximum">
            <number>5000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_getROI">
           <property name="text">
            <string>Get ROI</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_estimate">
         <property name="text">
          <string>Estimate</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtAddons::ImageViewerWidget</class>
   <extends>QWidget</extends>
   <header location="global">imageviewerwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PiercingPointDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PiercingPointDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
